db/migrate/20200830161328_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,21 @@
+MIGRATION_CLASS =
+  if ActiveRecord::VERSION::MAJOR >= 5
+    ActiveRecord::Migration["#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"]
+  else
+    ActiveRecord::Migration
+  end
```

Comment:
This is auto-generated by friendly_id. We could probably remove it, and just use L3.

Reply:
Yeah, you're not going to downgrade, ever.